.github/workflows/gradleBuild.yml

```
@@ -46,18 +46,18 @@ jobs:
 
       # Run assemble tasks
       - name: Run assemble tasks
-        uses: gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7
+        uses: gradle/gradle-build-action@0d13054264b0bb894ded474f08ebb30921341cee
```

Use version instead of commit id
```suggestion
        uses: gradle/gradle-build-action@v2.1.4
```